redux-optimistic-ui/redux-optimistic-ui.d.ts

```diff
@@ -0,0 +1,78 @@
+// Type definitions for redux-optimistic-ui 0.4.0
+// Project: https://github.com/mattkrick/redux-optimistic-ui
+// Definitions by: Alexey Svetliakov <https://github.com/asvetliakov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../redux/redux.d.ts" />
+
+declare module "redux-optimistic-ui" {
+    import { Reducer } from "redux";
+
+    /**
+     * Enhanced state by redux-optimistic-ui
+     */
+    export interface OptimisticState<TState> {
+        /**
+         * History list
+         * List is Immutable.List<>
+         */
+        history: any;
```

Comment:
I thought it's too earlier to introduce TS 2.0 features at least when the 2.0 is not even beta stage.